.github/workflows/ci.yml

```diff
@@ -11,28 +11,34 @@ jobs:
     name: Unit Tests
     runs-on: ubuntu-latest
     steps:
-      - name: Set up Go 1.x
-        uses: actions/setup-go@v2
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Go 1.x
+        uses: actions/setup-go@v3
         with:
-          go-version: 1.17.2
-        id: go
+          go-version: '>=1.17.2'
 
-      - name: Check out code into the Go module directory
-        uses: actions/checkout@v2
+      - name: Setup buf
+        uses: bufbuild/buf-setup-action@v1.3.1
+      
+      - name: Lint protos
+        uses: bufbuild/buf-lint-action@v1
+        with:
+          input: 'api-spec/protobuf'
 
       - name: Run Gosec Security Scanner
-        run: |
-          export PATH=$PATH:$(go env GOPATH)/bin
-          curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v2.9.3
-          gosec -severity high -quiet ./...
+        uses: securego/gosec@master
+        with:
+          args: '-severity high -quiet ./...'
```

Comment:
we used the other way since wasn't not working somehow (ie. reporting the security high)